### PR TITLE
fix(Examples): MSDK-1194 Fix to select single SPI operating modes for all Micros

### PR DESCRIPTION
**Pull request by Suraj-Ajjampur**
As highlighted in [MSDK-1194](https://jira.maxim-ic.com/browse/MSDK-1194), there are 3 operating modes in the SPI example of the MAX78000. This fix ensures that a single operating mode is supported by the compiler by making changes to some pre-processor directives and the underlying logic.

### CODE REVIEW

**Review by Jake-Carter — APPROVED** [review]

**Comment by Jake-Carter** [comment]
> See comment in [PR #622](https://github.com/Analog-Devices-MSDK/msdk/pull/622) about the README changes. The SPI example changes look good though!

Uno reverse card https://github.com/Analog-Devices-MSDK/msdk/pull/622#issuecomment-1593280837

File: Examples/MAX32650/SPI/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This example configures the SPI to send data between the MISO (1.28) and MOSI (P
 
 Multiple word sizes (2 through 16 bits) are demonstrated.
 
-The example can be configured to perform blocking (synchronous), non-blocking (asynchronus), or DMA transactions by defining/selecting the MASTERSYNC, MASTERASYNC, or MASTERDMA macro respectively.
+The example can be configured to perform blocking (synchronous), non-blocking (asynchronus), or DMA transactions by setting the MASTERSYNC, MASTERASYNC, or MASTERDMA macro respectively.
```
Comment (Jacob-Scheiffler) [inline Examples/MAX32650/SPI/README.md:7]: Consider updating to match other READMEs.

**Review by Jacob-Scheiffler — CHANGES_REQUESTED** [review]

File: Examples/MAX32662/SPI/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ MOSI (P0.3) pins.  Connect these two pins together.
 
 Multiple word sizes (2 through 16 bits) are demonstrated.
 
-This example can demonstrate blocking, non-blocking, and DMA transactions methods controlled by the definitiona at the top of main. For blocking transactions define MASTERSYNC, for non-blocking transactions define MASTERASYNC, and for DMA transactions define MASTERDMA; define only one of these at a time. By default MASTERSYNC is defined.
+This example can demonstrate blocking, non-blocking, and DMA transactions methods controlled by the definitiona at the top of main. For blocking transactions define MASTERSYNC, for non-blocking transactions define MASTERASYNC, and for DMA transactions define MASTERDMA; set only one of these at a time. By default MASTERSYNC is set to 1.
```
[the remark this answers is not in the document]
Comment (Jacob-Scheiffler) [reply Examples/MAX32662/SPI/README.md:8]: @Suraj-Ajjampur I agree that it's not critical for this release, but these inconsistencies tend to stick around for a long time so it's best to make it consistent while you're thinking of it.

**Comment by Suraj-Ajjampur** [comment]
@Jacob-Scheiffler Changes made. Could you please approve and merge this?

**Review by Jacob-Scheiffler — APPROVED** [review]

**Pull request merged**